region/crateria/central.json

```diff
@@ -153,6 +204,32 @@
               ],
               "openEnd": 1
             }
+          ],
+          "leaveWithGMode": [
+            {
+              "leavesWithArtificialMorph": false,
+              "leavesWithOverloadedPLMs": true,
+              "strats": [
+                {
+                  "name": "Overload Scroll PLMs",
+                  "notable": false,
+                  "requires": [
+                    {"or": [
+                      "SpeedBooster",
```

Comment:
wouldnt this require one of many strats to break these blocks? shinesparking from the right, blue space jumps from any of the 3 locations, bombs, or PBs, etc? might be able to simplify this by using obstacle A.

Reply:
Bombs and PBs won't work to break the blocks in G-mode, and blue space jump requires an additional item (SpaceJump), so I don't think we lose anything significant by just considering these two options.

Reply:
I added some more requirements now to properly represent what is needed for the Big Jump short-charge from the plateau, which is a simplification to avoid considering all the other easier ways they could get through with the same items. If someone is enabling G-mode logic, I don't think we lose too much by also requiring basic things of lower difficulty/complexity.